ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -553,9 +558,11 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
         map(currentAddress => this.meetsActivityRequirements(currentAddress))
     );
 
-    const emitValidStatusAction$ = combineLatest([this.formErrorMessages$, this.verifyFieldErrors$, activitityRequirementsMet$]).pipe(
-        map(([formErrors, addressErrors, reqsMet]) =>
-            !formErrors.length && !addressErrors.length && reqsMet),
+    const emitValidStatusAction$ = combineLatest([this.formValidStatusChanged$, this.formErrorMessages$, this.verifyFieldErrors$,
+        activitityRequirementsMet$]).pipe(
+        map(([formValidStatusChanged, formErrors, addressErrors, reqsMet]) =>
+            (!this.configuration.addressEnforceRequiredFields || formValidStatusChanged)
+                && !formErrors.length && !addressErrors.length && reqsMet),
         distinctUntilChanged(),
```

Comment:
We care about the valid status for overall component valid status if configuration flag is set to true.